frontend/src/pages/FoodDiary.js

```diff
@@ -19,24 +19,21 @@ export default function FoodDiary() {
     getFood(token).then(data => setFoodList(data.foodList))
   }
 
-  const Items = foodList.map(Item => (
-    <ItemCard
-      valueTitle={'Food'}
-      mainValue={Item.foodName}
-      date={Item.date}
-      id={Item.id}
-      key={Item.id}
-      reloadList={reloadFoodList}
-      editService={updateFood}
-      removeService={removeFood}
-    />
-  ))
-
   return (
     <Wrapper>
       <Header title="Food Diary" />
       <Content>
-        <ul>{Items}</ul>
+        <ul>
```

Comment:
I think the `ul` tag belongs inside `ItemGallery`

Check you `ItemCard` component! It has a div as a wrapper with a li inside. Only li are allowed as direct children of ul. Refactor the wrapper to be li ;)